tasks/main.yml

```
@@ -4,14 +4,20 @@
 
 - include_tasks: "setup-{{ ansible_distribution|lower }}.yml"
 
+- name: Load packages variable file based on the OS type, or a default if not found
+  include_vars: "{{ lookup('first_found', params) }}"
+  vars:
+    params:
+      files:
+        - "packages-{{ ansible_distribution }}.yml"
```

```suggestion
        - "packages-{{ ansible_distribution|lower }}.yml"
```
This way `vars/packages-Archlinux.yml` can be renamed to `vars/packages-archlinux.yml`.